tests/linearizability/model.go

```diff
@@ -240,16 +240,7 @@ func initState(request EtcdRequest, response EtcdResponse) EtcdState {
 			state.KeyValues[op.Key] = op.Value
 		case Delete:
 		case PutWithLease:
-			if _, ok := state.Leases[op.LeaseID]; ok {
-				state.KeyValues[op.Key] = op.Value
-				//detach from old lease id but we dont expect that at init
-				if _, ok := state.KeyLeases[op.Key]; ok {
-					panic("old lease id found at init")
-				}
-				//attach to new lease id
-				state.KeyLeases[op.Key] = op.LeaseID
-				state.Leases[op.LeaseID].Keys[op.Key] = leased
-			}
+			//nop here since lease wont be there
```

Comment:
We already support multiple operations, so in theory it's possible that `LeaseGrant` happens before `PutWithLease`.  But current implementation doesn't support standalone `LeaseGrant`, which is executed along each `PutWithLease`, so it's OK for now.